media-library/overview/search-update-and-delete.md

```diff
@@ -67,6 +67,18 @@ There are two places where you can add a new search query -
 1. Clicking on the `+` icon next to an existing filter lets you combine other search parameters in the same filter.
 2. Clicking on `+ Add Filter` link lets you add another filter in parallel to the existing one. This new filter can be combined either to narrow (`and` operator) or to expand (`or` operator) the results from other search filters.
 
+### Visual Search
+
+{% hint style="info" %}
+**Enterprise plan only**
+
+This feature is only available in custom enterprise pricing plans.
+{% endhint %}
+
+Visual search allows you to search for similar images based on visual content. Instead of relying on text-based keywords or metadata, you can search based on visual content like colors, shapes, textures and any other information that can be derived from the image itself.
```

Comment:
Isn't searching for context "summer dress" also text-based keyword?

Reply:
that's a part of any other information